src/presentational/NewClassroomFormElement.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+const NewClassroomFormElement = props => {
+  const { label, ...other } = props;
+  other.name = other.name || label.toLowerCase();
+
+  return (
+    <div className="form-group">
+      <label>{ label }</label>
```

Comment:
Knackers the styling :( And IDs don't work, as they're getting broken by react-a11y. Presumably that's disabled in production?